extensions/cornerstone/src/init.tsx

```diff
@@ -179,7 +179,10 @@ export default async function init({
   initWADOImageLoader(userAuthenticationService, appConfig, extensionManager);
 
   /* Measurement Service */
-  this.measurementServiceSource = connectToolsToMeasurementService(servicesManager);
+  this.measurementServiceSource = connectToolsToMeasurementService({
```

Comment:
This will have to be documented in the migration guide.
Also, my preference is to just pass in the services object so that anyone needing any of the core services just gets the services, rather than having anyone "know" about what needs to be passed in.